### ISSUE TITLE
Reproducing the CoNLL2003 Results

### ISSUE THREAD

**Issue by F9GK** [cjymz886/GFNER#3]
Hi, I cannot reproduce the CoNLL2003 dataset results for f1=93.85 in your paper; the results only reached f1=91.

The parameters and model I used are from your code.:

    vocab_size=30290
    embedding_size=128
    seq_len=127
    random_seed = 2009
    hidden_dropout_prob=0.1
    learning_rate=3e-5
    weight_decay=0.0
    max_grad_norm=1.0
    warmup=0.0
    fix_bert_embeddings=False
    min_num=1e-7
    use_bert = True
    batch_size=6
    epochs=50
    device = torch.device("cuda")

    #bert_model = 'biobert-base-cased-v1.2'
    #bert_model='biobert-large-cased-v1.1'
    bert_model = 'bert-large-cased'

If available, could you provide me with a detailed list of the parameters you used for training?
Thank you !
